package.json

```diff
@@ -4,7 +4,10 @@
   "author": "Daniel Wirtz <dcode@dcode.io>",
   "description": "The swiss army knife for binary data in JavaScript.",
   "main": "index.js",
-  "browser": "dist/ByteBufferAB.js",
+  "browser": {
+    "index.js": "bytebuffer/dist/ByteBufferAB.js",
+    "Long": "bytebuffer/node_modules/long/dist/Long.js"
```

Comment:
Thanks @englercj, the issue I was trying to address is getting webpack to use `node_modules/long` as the provider of `Long`.